ptsprojects/zephyr/l2cap.py

```diff
@@ -129,6 +130,27 @@ def test_cases(pts):
                       TestFunc(stack.l2cap_init, le_psm, le_initial_mtu),
                       TestFunc(btp.l2cap_le_listen, le_psm)]
 
+
+    pre_conditions_eatt = [TestFunc(btp.core_reg_svc_gap),
```

Comment:
Again, see if we can use a common pre_conditions list and then add eatt specific stuff to it.